modules/serialize/test/deserialize_factory.test.ts

```diff
@@ -78,6 +78,29 @@ describe('deserializeFactory:decodeEncryptionContext', () => {
     expect(test).to.have.property('information')
       .and.to.eql('\u00bd + \u00bc = \u00be')
   })
+
+  it('Keys may be properties of Object.prototype, decodeEncryptionContext has to succeed', () => {
+    const { decodeEncryptionContext } = deserializeFactory(toUtf8, WebCryptoAlgorithmSuite)
+
+    /* Single key-value pair */
```

Comment:
Can you move this to the `fixtures.ts` file?  This is a good test vector to have, and is a little more portable :)

Reply:
Yes, I have added the new test vector in `fixtures.ts` and adjusted the tests accordingly.